src/Appwrite/Utopia/Database/Validator/Query.php

```diff
@@ -1,21 +0,0 @@
-<?php
-
-namespace Appwrite\Utopia\Database\Validator;
-
-use Utopia\Database\Validator\UID;
-use Utopia\Database\Validator\Query as QueryValidator;
-
-class Query extends QueryValidator
-{
-    protected function isValidCursor($cursor): bool
-    {
-        $validator = new UID();
-
```

Comment:
Looked like this class was performing the same validation as a cursor validator. Hence removed it in favour of Validator\Query\Cursor.php